Cargo.lock

```diff
@@ -1985,9 +1985,9 @@ checksum = "5a9d968042a4902e08810946fc7cd5851eb75e80301342305af755ca06cb82ce"
 
 [[package]]
 name = "indexmap"
-version = "1.8.0"
+version = "1.8.1"
```

Comment:
Unreverted this version bump because cargo wouldn't keep it reverted. Release notes look sane: https://github.com/bluss/indexmap/blob/master/RELEASES.md